langchain/prompts/chat.py

```diff
@@ -141,7 +149,15 @@ def format_messages(self, **kwargs: Any) -> List[BaseMessage]:
 
 class ChatPromptTemplate(BaseChatPromptTemplate, ABC):
     input_variables: List[str]
-    messages: List[Union[BaseMessagePromptTemplate, BaseMessage]]
+    messages: List[
+        Union[
+            BaseMessagePromptTemplate,
+            BaseMessage,
+            AIMessagePromptTemplate,
```

Comment:
all the message prompt templates inherit from BaseMessagePromptTemplate, so not needed right?